### PR TITLE
For release

### PR DESCRIPTION
**Pull request by Sutrannu**

### CODE REVIEW

File: generator/source/release/bumper/notes.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+
+**Note**: Requires Docker Engine 18.02.0 or later.
+
+## Kubernetes
+
+- This release includes example files for two Kubernetes deployments: A
+  five-node network using PoET simulator and a single-node environment using
+  dev mode consensus. The Application Developer’s Guide describes how to use
+  these example files; see the [Hyperledger Sawtooth
+  documentation](#hyperledger-sawtooth-documentation).
```
Comment (annechenette) [inline generator/source/release/bumper/notes.md]: Bad link.  It should be this link instead:

https://sawtooth.hyperledger.org/docs/core/nightly/master/app_developers_guide.html

Comment (annechenette) [reply generator/source/release/bumper/notes.md]: Except use the right "latest" version of that link.  Sorry!

Comment (Sutrannu) [reply generator/source/release/bumper/notes.md]: Link href set to https://sawtooth.hyperledger.org/docs/core/releases/latest/app_developers_guide.html

File: generator/source/release/bumper/notes.md [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+- [Website](#website): Includes the Sawtooth website content, structure, and
+  website development processes
+
+# Core Sawtooth Components
+
+## New Features
+
+- The consensus API has been completely redesigned. Consensus has been moved to
+  a separate process, called a “consensus engine”. See [Hyperledger Sawtooth
+  Consensus](#hyperledger-sawtooth-consensus) for more details.
```
Comment (annechenette) [inline generator/source/release/bumper/notes.md]: This is supposed to link with the "Consensus" section lower down (line 113).

`# Consensus`

**Review by annechenette — APPROVED** [review]

**Review by rberg2 — APPROVED** [review]

**Pull request merged**